src/Microsoft.Sbom.Api/Config/ConfigSanitizer.cs

```diff
@@ -147,6 +150,36 @@ private ConfigurationSetting<IList<ManifestInfo>> GetDefaultManifestInfoForValid
             };
         }
 
+    private ConfigurationSetting<IList<ManifestInfo>> GetDefaultManifestInfoForGenerationAction(IConfiguration configuration)
```

Comment:
We should have UT coverage for this code. See ConfigSanitizerTests.cs, lines 94-112 for the Validation counterparts